push-api/JSONParser/Parser.py

```diff
@@ -16,13 +20,13 @@ def readDocument(config):
             documentArray = data[arrayKey]
             Parser.parseArray(documentArray, organisation, apiKey, sourceName, urlValue)
 
+# Push array in source
     def parseArray(array, organisation, apiKey, sourceName, urlValue):
+        StaReb = 'REBUILD'
```

Comment:
What about putting those constants in their own file so that you can use them like this:

```
import SourceStatus;
SourceStatus.REBUILDING
SourceStatus.IDLE
```